code-of-conduct.html

```diff
@@ -170,11 +170,11 @@ <h2>CONTACT INFORMATION</h2>
           <div class="col-lg-3 col-md-6 footer-links">
             <h4>Useful Links</h4>
             <ul>
-              <li><i class="bi bi-chevron-right"></i> <a href="#">Home</a></li>
+              <li><i class="bi bi-chevron-right"></i> <a href="index.html">Home</a></li>
               <li><i class="bi bi-chevron-right"></i> <a href="#">About us</a></li>
               <li><i class="bi bi-chevron-right"></i> <a href="#">Services</a></li>
               <li><i class="bi bi-chevron-right"></i> <a href="#">Terms of service</a></li>
-              <li><i class="bi bi-chevron-right"></i> <a href="#">Privacy policy</a></li>
+              <li><i class="bi bi-chevron-right"></i> <a href="privacy-policy.html">Privacy policy</a></li>
```

Comment:
@basiima why are we adding routes to pages that dont exist?
![image](https://user-images.githubusercontent.com/65954740/225571228-6698c8c7-1067-45cf-b1c1-a285a98358aa.png)

Reply:
for this route to be avaialable...the `privacy-policy.html` file needs to be available

Reply:
The privacy policy page is created in a separate pull request.

Reply:
alright then, could we please merge that pull request first?

Reply:
so i think we are good here since #7 is merged